docs(observability): how to deactivate cockpit (#922)
scaleway/docs-content: observability/cockpit/how-to/deactivate-cockpit.mdx

---
meta: How to deactivate your Cockpit
  title: How to deactivate your Cockpit
  description: This page shows you how to deactivate your Cockpit
content:
  h1: How to deactivate your Cockpit
  paragraph: This page shows you how to deactivate your Cockpit
categories:
  - observability
dates:
  validation: 2022-10-17
  posted: 2022-10-30
---

This page shows you how to deactivate your Cockpit.

<Message type="important">

 By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted.

</Message>

<Message type="requirement">

  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)

</Message>

> 1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.

Reviewer: ```suggestion
1. Click **Cockpit** in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

---
meta: How to deactivate your Cockpit
  title: How to deactivate your Cockpit
  description: This page shows you how to deactivate your Cockpit
content:
  h1: How to deactivate your Cockpit
  paragraph: This page shows you how to deactivate your Cockpit
categories:
  - observability
dates:
  validation: 2022-10-17
  posted: 2022-10-30
---

This page shows you how to deactivate your Cockpit.

<Message type="important">

 By deactivating your Cockpit you will no longer be able to monitor your Scaleway resources, configure the sending of external metrics and logs, or receive alerts. All associated data will be permanently deleted.

</Message>

<Message type="requirement">

>   - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)

Author: ```suggestion
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
```

  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)

</Message>

> 1. Log into your [Scaleway Console](https://console.scaleway.com/) account.

Reviewer: ```suggestion
1. Log into your [Scaleway console](https://console.scaleway.com/) account.
```

Reviewer: Actually I don't think you need this step because it's already in the requirements. Maybe edit step 2 to be "Click **Cockpit** in the Observability section of the console side menu..."

> 1. Log into your [Scaleway Console](https://console.scaleway.com/) account.

Author: ```suggestion
```

> 2. Click **Cockpit** in the side menu. The **Cockpit** overview page displays.

Author: ```suggestion
1. Click Cockpit in the Observability section of the [console](https://console.scaleway.com/) side menu. The **Cockpit** overview page displays.
```

> 3. Click the **Settings** tab.

Author: ```suggestion
2. Click the **Settings** tab.
```

> 4. Click **Deactivate Cockpit**.

Author: ```suggestion
3. Click **Deactivate Cockpit**.
```

> 5. Type **DEACTIVATE** and then click **Deactivate Cockpit** to confirm. Your Cockpit will be deleted in a matter of minutes.

Author: ```suggestion
4. Type **DEACTIVATE** and then click **Deactivate Cockpit** to confirm. Your Cockpit will be deleted in a matter of minutes.
```
